### ISSUE TITLE
Manual flow PyPS

### ISSUE THREAD

**Issue by volter29** [kylemurray2/PyPS#2]
Hi and thanks for sharing these scripts with us.
is it possible to have a manual for the processing of PYPS? 

Thanks

FM

**Comment by kylemurray2** [comment]
Yes, I am changing the code quite a bit right now, but I should have a new Readme file in the next couple of weeks.

**Comment by volter29** [comment]
thanks a lot  Kyle!

**Comment by jackamazing** [comment]
I also want the PSPY processing manual, but it is not detailed in Readme file.
